src/DataFrames.jl

```diff
@@ -92,6 +92,8 @@ export @~,
 ##
 ##############################################################################
 
+const displaysize(io::IO) = Compat.displaysize()
```

Comment:
Yeah, I realized that after saying it. I wish I knew what was wrong with my brain today...